README.md

```diff
@@ -95,25 +95,32 @@ Sign up at [hCaptcha](https://www.hcaptcha.com) to get your sitekey today. Check
 
 ![Demo](https://raw.githubusercontent.com/hCaptcha/vue-hcaptcha/master/screenshots/demo.gif)
 
-To run the demo, simply ```npm run serve``` after downloading the repo, this will start a development server on localhost:8080. Open your console to see the demo app emitting events.
-
-To test locally, you can run the following NPM commands:
-- ```npm run serve```
-    - This initiates the hot reload dev server from the vue-cli
-- ```npm run build```
-    - This will build your version of the component with your customizations for production.
-- ```npm run lint``` and ```npm run lint:fix```
-    - This will enable ESLint to help keep your code clean!
+To run the demo:
+1. clone this repo `git clone https://github.com/hCaptcha/vue-hcaptcha.git`
+2. run ```npm run serve:demo:traditional``` 
+   * it will start the demo app on localhost:8080
+   * open your console to see the demo app emitting events
 
 ### TypeScript
 
-Please see the [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44299) entry.
+TypeScript is supported (`types/index.d.ts`), and you can see an example by running `npm run serve:demo:ts`.
+
+
+### Note for maintainers
+
+### Scripts
+
+* `npm run serve:demo:traditional` - will start the traditional demo app with hot reload
```

Comment:
Could this and the ts be shortened to: 
- traditional > `npm run serve` or `npm run serve:traditional`
- ts > `npm run serve:ts`

Traditional seems a bit verbose and not needed, just my thought, fine either way.

Reply:
Agree. Fixed.